contrib/ci/dependencies.xml

```diff
@@ -1596,6 +1660,7 @@
     </distro>
     <distro id="ubuntu">
       <package variant="x86_64" />
+      <package variant="20.04" />
```

Comment:
@mkopec Why do we need this change and it was not presetn in the usptream? Is it upstreamable?

Reply:
fwupd upstream doesn't build packages for ubuntu. I tried adding a new variant `ubuntu-20.04` with a different CI script for ubuntu, but at the moment this code is unused.

Reply:
@mkopec Well, I think it **is** used.
Here: https://github.com/Dasharo/fwupd/blob/dasharo-ci/contrib/ci/debian.sh#L34
Then here: https://github.com/Dasharo/fwupd/blob/dasharo-ci/contrib/ci/generate_debian.py#L36

Reply:
Ok, maybe this `20.04` does not make much sense. But we need to provide `OS=ubuntu-x86_64` as a variable, so the `contrib` is generated correctly.